src/api/cluster_server_api.js

```diff
@@ -162,6 +162,7 @@ module.exports = {
             auth: {
                 account: false,
                 system: false,
+                anonymous: true,
```

Comment:
Gap: in `read_server_config()` remove the call to `_get_aws_owner()` if possible or fix otherwise to avoid anonymous calls with unwanted side effects

Reply:
FE uses it, so I will not remove it yet.